packages/gatsby-plugin-image/README.md

```diff
@@ -155,15 +110,15 @@ module.exports = {
 
 ### API
 
-The only required prop is `src`. The default type is `fixed`. The other props match those of [the new GatsbyImage component](#gatsby-image-next-generation)
+The only required prop is `src`. The default type is `fixed`. The other props match those of [the new GatsbyImage component](#gatsbyimage). You can also pass in options which are forwarded to [`gatsbyImageData`](#graphql-resolver).
 
-## gatsby-image next generation
+## GatsbyImage
 
 Speedy, optimized images without the work.
 
-gatsby-image is a React component specially designed to give your users a great image experience. It combines speed and best practices. You can use any image processing library that you want. We suggest using gatsby-plugin-sharp as your image processor. Saving images locally improves [the important health metrics](https://web.dev/vitals/) for your site.
+gatsby-image is a React component specially designed to give your users a great image experience. It combines speed and best practices.
```

Comment:
```suggestion
GatsbyImage is a React component specially designed to give your users a great image experience. It combines speed and best practices.
```
This is tough but I might recommend this so users aren't confused with the previous `gatsby-image` package.